docs/src/index.md

```diff
@@ -19,8 +23,12 @@ These all use intermittent energy sources in the production of energy, so the ma
 
 ### [`HydroStor`](@ref) and [`PumpedHydroStor`](@ref)
 
-The other nodes implement a regulated hydropower storage plant, both with ([`PumpedHydroStor`](@ref)) and without pumps ([`HydroStor`](@ref)) for filling the reservoir with excess energy.
-The hydropower storage plant can also be extended as they are declared as subtypes of [`HydroStorage`](@ref).
+The [`PumpedHydroStor`](@ref) and [`HydroStor`](@ref) nodes implement a regulated hydropower storage plant, either with or without pumps for filling the reservoir with excess energy.
```

Comment:
Kan vi spesifisere litt mer om forskjellen? Feks: "These nodes can be used to model a single hydropower plant and reservoir, or to model an aggregated description of a hydropower system. The nodes do not include conversion from water to energy, and therby requires an energy-based descriton of the hydropower system."

Reply:
Vi kan jo vurdere å gjøre det senere. Det er jo ikke så viktig nå, og jeg er heller ikke 100% sikker på om det er helt rett.

Reply:
Synes det var et godt forslag, kan legge det inn.